Assets/HoloToolkit/Utilities/Scripts/ManualCameraControl.cs

```diff
@@ -28,6 +28,18 @@ public enum MouseButton
     public MouseButton MouseLookButton = MouseButton.Shift;
     public bool IsControllerLookInverted = true;
 
+    public enum ControlMode
```

Comment:
It does look like there's quite the mix of being included in a namespace and not. My vote is that we should unless there's a compelling reason not to. Maybe hold off on it for  now and make an issue pointing out this inconsistency. That way if there's some reason for it there's a place for that to come up.